test/local.mk

```diff
@@ -134,4 +134,5 @@ test-e2e: $(kuttl_bin) $(mc_bin) local-install provider-config ## E2E tests
 	@if [ -f $(KIND_KUBECONFIG) ]; then kubectl delete buckets --all; else echo "no kubeconfig found"; fi
 	@if [ -f $(KIND_KUBECONFIG) ]; then kubectl delete iamkeys --all; else echo "no kubeconfig found"; fi
 	@if [ -f $(KIND_KUBECONFIG) ]; then kubectl delete postgresql --all; else echo "no kubeconfig found"; fi
+	@if [ -f $(KIND_KUBECONFIG) ]; then kubectl delete mysql --all; else echo "no kubeconfig found"; fi
```

Comment:
oh, I forgot that change for redis. Will add it, thanks for the tip ;)

Reply:
oh, I didn't realized that you forgot it, else I'd have told you 😄